dockerfiles/QAT/README.md

```diff
@@ -0,0 +1,59 @@
+
+  * [Introduction](#introduction)
+  * [Building](#building)
+  * [Options](#options)
+
+## Introduction
+
+The files in this directory can be used to build a modified Kata Containers rootfs
+and kernel with modifications to support Intel QAT hardware.
+
+The generated files will need to be copied and configured into your Kata Containers
+setup.
+
+Please see the
+[Kata QAT documentation](https://github.com/kata-containers/documentation/blob/master/use-cases/using-Intel-QAT-and-kata.md)
+for more specific QAT details.
+
+## Building
+
+The process is executed using Docker:
+
+```sh
+$ cd ..
+$ docker build --label kataqat --tag kataqat:latest QAT
+$ mkdir ./output
+$ docker run -ti --rm --privileged -v /dev:/dev -v $(pwd)/output:/output kataqat
+```
+
+> **Note:** The use of the `--privileged` and `-v /dev:/dev` arguments to the `docker run` are
```

Comment:
so, the will not work in a kata container?

Reply:
probably correct - just like osbuilder will not afaik - it does the same thing in `USE_DOCKER` mode I think